Makefile

```diff
@@ -130,14 +133,18 @@ ami-check: generate-ami ## Check whether the AMIs have been updated and fail if
 
 .PHONY: generate-kubernetes-types
 generate-kubernetes-types: ## Generate Kubernetes API helpers
-	@build/vendor/k8s.io/code-generator/generate-groups.sh deepcopy,defaulter _ github.com/weaveworks/eksctl/pkg/apis eksctl.io:v1alpha5
+	@go mod download k8s.io/code-generator # make sure the code-generator is present
+	@echo "/*\n$$(cat LICENSE)*/\n" > codegenheader.txt
```

Comment:
What is this for?

Reply:
It's the header of the generated files, without providing an explicit header (`--go-header-file` ) `generate-groups.sh` doesn't work with go modules (`k8s.io/code-generator` doesn't fully accept go modules).

Reply:
Ah, I see. Would you mind to add a comment to clarify this?

Reply:
Done

Reply:
Also, would you mind calling it `.kubernetes-code-generator-headers`? So it not too visible.

Reply:
I think it's better for it to be explicit